### ISSUE TITLE
`ctf_stats:good_weapons` list requires immediate attention

### ISSUE THREAD

**Issue by ClobberXD** [rubenwardy/capturetheflag#110]
[from ctf_stats mod - init.lua]
/**************************************/
local good_weapons = {
	"default:sword_stone",
	"default:sword_steel",
	"shooter:grenade",
	"shooter:shotgun",
	"shooter:rifle",
	"shooter:machine_gun",
}
/**************************************/

The `ctf_stats:good_weapons` list is used to calculate the points for a kill, where the **points are halved if the enemy does not have "good weapons" at the time of death**. But the list does not seem to include bronze, mese and diamond implements, meaning that even if a player manages to kill a pro carrying a mese sword, their kill score is halved.

Also, the stone sword is considered a 'good weapon' only at the hands of a couple of lethal pros, and is generally not a good weapon IMO.

**Comment by afshaan4** [comment]
>Also, the stone sword is considered a 'good weapon' only at the hands of a couple of lethal pros, and is generally not a good weapon IMO.

So true

**Comment by AKryukov92** [comment]
I want to note, that all other kinds of sword are not present in this list. I mean bronze, mese, diamond. They are possible in the Cave map.

**Comment by ClobberXD** [comment]
Yes - that's exactly what I mentioned - imagine getting your points halved just because the enemy did not have a steel sword (in spite of them having a mese sword)

**Comment by afshaan4** [comment]
I also think you should add steel pick to the list, in the "two hills" and "bridge" they are quite lethal.

**Comment by AKryukov92** [comment]
Steel pick is actually better than stone sword. With same damage, you can hit enemy often (for 0.2 sec/hit)
Though, I think it is not enough to consider it as "good weapon".
Look https://github.com/rubenwardy/capturetheflag/blob/master/mods/default/tools.lua
at properties "full_punch_interval" and "fleshy"
```
minetest.register_tool("default:pick_steel", {
	description = "Steel Pickaxe",
	inventory_image = "default_tool_steelpick.png",
	tool_capabilities = {
		full_punch_interval = 1.0,
		max_drop_level=1,
		groupcaps={
			cracky = {times={[1]=4.00, [2]=1.60, [3]=0.80}, uses=20, maxlevel=2},
		},
		damage_groups = {fleshy=4},
	},
	sound = {breaks = "default_tool_breaks"},
})
```
```
minetest.register_tool("default:sword_stone", {
	description = "Stone Sword",
	inventory_image = "default_tool_stonesword.png",
	tool_capabilities = {
		full_punch_interval = 1.2,
		max_drop_level=0,
		groupcaps={
			snappy={times={[2]=1.4, [3]=0.40}, uses=20, maxlevel=1},
		},
		damage_groups = {fleshy=4},
	},
	sound = {breaks = "default_tool_breaks"},
})
```

**Comment by ClobberXD** [comment]
If the steel pick is added to the list, every player has 'good weapons' in the cavern match... :)

**Comment by afshaan4** [comment]
Oh yeah, ok.
